.github/workflows/autoupdate-prod.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Release updates
         if: steps.verify-changed-files.outputs.files_changed == 'true'
         working-directory: ./tools
-        run: make new-release  
+        run: make new-version  
```

Comment:
Missing documentation update across the repository.

Reply:
there was not any previous doc for new-release, I've updated the Makefile goal to make it similar to the script it executes. No occurrences in main branch for string "new-release"